src/index.ts

```diff
@@ -0,0 +1,17 @@
+export class Challenge {
+    num : number;
```

Comment:
This is unused, remove it.
For the future: remember to mark members always as `public`, `private` or `protected`. For convention non-public members have a leading underscore. 
For example: `public num: number;` and `private _num: number;`